src/s1cmp.hbs

```diff
@@ -75,6 +79,12 @@
 			}
 		});
 
+		cmp('addEventListener', 'onSubmit', () => {
+			if (window.ReactNativeWebView && window.ReactNativeWebView.postMessage) {
```

Comment:
it may be worth going ahead and abstracting this up to a function
```
const nativePostMessage = (window.ReactnativeWebView && window.ReactNativeWebView.postMessage) ? (namespace, msg) => {
    window.ReactNativeWebView.postMessage(namespace, msg);
} : () => {
  // no-op
}